config/locales/ja.yml

```
@@ -1,22 +1,24 @@
-book_info: &book_info
-  title: "題名"
-  memo: "メモ"
-  author: "著者"
-  picture: "写真"
+ja:
+  page_title:
+    index: "本"
+    create: "新規の本"
+    edit: "本の編集"
 
-button: &button
-  show: "表示"
-  edit: "編集"
-  destroy: "削除"
-  newbook: "新しい本の追加"
-  back: "戻る"
+  book_info:
+    title: "題名"
+    memo: "メモ"
+    author: "著者"
+    picture: "写真"
 
-message: &message
-  finalconfirmation: "本当によろしいですか？"
-  create_info: "本の情報は作成されました"
-  update_info: "本の情報は更新されました"
-  destroy_info: "本の情報は削除されました"
-ja:
-  book_info: *book_info
-  button: *button
-  message: *message
+  button:
+    show: "表示"
+    edit: "編集"
+    destroy: "削除"
+    newbook: "新しい本の追加"
```

```suggestion
    new: "新しい%{name}の追加"
```

のように翻訳の一部をパラメータ化すると翻訳を再利用しやすくなります。